application/Modules/Invoices/Models/Item.php

```
@@ -145,7 +172,7 @@ public function delete($item_id): bool
         $this->db->where('item_id', $item_id);
         $this->db->delete('ip_invoice_item_amounts');
 
-        $this->load->model('invoices/mdl_invoice_amounts');
+        $this->load->model('invoices/invoice_amount');
         $global_discount['item'] = $this->mdl_invoice_amounts->get_global_discount($invoice_id);
```

PHPMD (2.15.0)

Avoid using undefined variables such as '$global_discount' which will lead to PHP notices. (undefined)

(UndefinedVariable)